tests/js/spec/components/hook.spec.jsx

```diff
@@ -3,37 +3,43 @@ import {render, screen} from 'sentry-test/reactTestingLibrary';
 import Hook from 'sentry/components/hook';
 import HookStore from 'sentry/stores/hookStore';
 
-describe('Hook', function () {
-  const Wrapper = function Wrapper(props) {
-    return <div data-test-id="wrapper" {...props} />;
-  };
-
-  beforeEach(function () {
-    HookStore.add('footer', ({organization} = {}) => (
-      <Wrapper key="initial" organization={organization}>
-        {organization.slug}
-      </Wrapper>
-    ));
-  });
+const HookWrapper = props => (
+  <div data-test-id="hook-wrapper">
+    {props.children} {JSON.stringify(props?.organization ?? {}, null, 2)}
+  </div>
+);
 
+describe('Hook', function () {
   afterEach(function () {
-    // Clear HookStore
+    HookStore.teardown();
     HookStore.init();
```

Comment:
Thoughts on having a `reinit` method that that be used for testing?

Reply:
I think I prefer an explicit init and teardown, they correspond better with the beforeEach and afterEach patterns imo. I think reinit could be confusing and indicate that it should be used in a beforeEach